RELEASING.md

```diff
@@ -59,3 +50,8 @@ Publish the release
   * In our case, ```comgoogleapi```
 * Click the *close*, then *release* button just below the top tabs
 * It will take some time (up to 10 minutes) for the package to transition
```

Comment:
We should be able to remove the manual instructions here since finalizeRelease should be able to automatically release and drop the staging repo for us. (so we don't need to log into the sonatype website at all)